webauthn/registration.go

```diff
@@ -41,18 +41,12 @@ func (webauthn *WebAuthn) BeginRegistration(user User, opts ...RegistrationOptio
 
 	credentialParams := defaultRegistrationCredentialParameters()
 
-	rrk := false
-	authSelection := protocol.AuthenticatorSelection{
-		RequireResidentKey: &rrk,
```

Comment:
Is it my understanding correct that the RF confing value for RequireResidentKey was also ignored previously?

Reply:
Indeed, the `AuthenticatorSelection` object in general.